src/main/java/webserver/HandlerMapping.java

```diff
@@ -0,0 +1,35 @@
+package webserver;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import webserver.request.HttpRequest;
+import webserver.requesthandler.*;
+
+import java.io.IOException;
+
+public class HandlerMapping {
```

Comment:
```suggestion
public class HandlerMapper {
```
영어권에서는 ing를 잘 사용하지 않아요. 대부분 한국식 표현입니다.